repository-data/application/src/main/resources/hcm-config/targeting/datastores/targetingdata.yaml

```diff
@@ -3,3 +3,5 @@ definitions:
   config:
     /targeting:targeting/targeting:datastores/targeting:targetingdata:
       maxAgeDays: 60
+    /targeting:targeting/targeting:datastores/targeting:visits:
+      targeting:storefactoryclass: com.onehippo.cms7.targeting.storage.elastic5.ElasticStoreFactory
```

Comment:
Enforces elastic search datastore to use Elastic search 5 - the version installed on the servers.